frontend/src/components/pages/status.tsx

```diff
@@ -153,7 +176,8 @@ export const Status = () => {
                   , but you <b>declined</b> your spot
                 </>
               )}
-              .
+              . Please let us know if you can not make it. And hand over your
```

Comment:
Consider changing the wording to "Please let us know if you can not make it so that we can hand over your spot to someone else."